README.md

```diff
@@ -174,6 +174,7 @@ APIs
 - [Jokes](https://jokes.one/api/joke/) - Full featured Jokes API 💸
 - [Perfect Tense API](https://www.perfecttense.com/developers) - The Perfect Tense API is the only spelling and grammar checking API that uses artificial intelligence to automatically correct all of your text in one call.💸
 - [qKast Channel Content](https://github.com/egfx/qKast) - Access live content collections sourced from any page around the web.
+- [Random Data Generator](https://randommer.io/randommer-api) - API Generator for  telephones, text, numbers, passwords, names, social security numbers, cards 💸
```

Comment:
Minor nit: Can we remove the extra whitespace before telephones? Also, can we remove the comma and put an "and" before cards?

Reply:
Done

Reply:
Thanks!